pcre2-sys/build.rs

```diff
@@ -85,6 +85,8 @@ fn main() {
     {
         builder.debug(true);
     }
+    // Enable c99 mode to compile pcre2 for issue #33
```

Comment:
Can you put a full link to the issue please? (And make sure comments are wrapped to 79 columns, inclusive.)

I'd also like to see this comment expanded a little bit. As it stands now, it's just repeating what the code says, which is effectively useless. So something like this instead:

> We enable C99 mode when compiling PCRE2 because without it, build errors have been observed in some environments. It's not totally clear why this only occurs in some environments, but one possible guess is that this supports older compilers. Either way, there is still some uncertainty about why enabling this flag is necessary. See https://github.com/BurntSushi/rust-pcre2/issues/33 for more details.

Reply:
> Can you put a full link to the issue please? (And make sure comments are wrapped to 79 columns, inclusive.)
> 
> I'd also like to see this comment expanded a little bit. As it stands now, it's almost just repeating what the code says, which is effectively useless (other than the issue reference). So something like this instead:
> 
> > We enable C99 mode when compiling PCRE2 because without it, build errors have been observed in some environments. It's not totally clear why this only occurs in some environments, but one possible guess is that this supports older compilers. Either way, there is still some uncertainty about why enabling this flag is necessary. See #33 for more details.

Thank you for your review. I've written full comments for it.